src/shared.rs

```diff
@@ -14,6 +14,16 @@ pub fn get_argument_names(
     args.keys().map(String::from).collect()
 }
 
+#[allow(clippy::implicit_hasher)]
```

Comment:
Nit: I don't think this allow is necessary since this works over a BTreeMap, not Hashmap :S